mkdocs.yml

```diff
@@ -22,6 +22,7 @@ nav:
   - Server:
       - Configuration: server/configuration.md
       - Auth: server/auth.md
+      - Google Auth: server/google-auth.md
```

Comment:
Oh, could you remove this, we purposely hid this one because this is not really a Unity Catalog feature - just an easy way to apply Google Auth if you don't already know.  We will want to add instructions to Okta and others - when we do , potentially we can have those as a submenu under the Auth.

Reply:
Oh, i don't know about this. In Action dashboard, this issue is showing that's why i added. But now, i removed this in my latest commit. 
Thanks!